client/pages/queue/[section].vue

```diff
@@ -84,8 +84,8 @@ const tabs: Tab[] = [
 const deadlineCol = {
   key: 'deadline',
   label: 'Deadline',
-  field: 'deadline',
-  format: (val: any) => val ? DateTime.fromISO(val).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY) : '',
+  field: 'externalDeadline',
+  format: (val: any) => val ? DateTime.fromJSDate(val as Date).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY) : '',
```

Comment:
`as` casting is a teensy bit risky as it can hide errors... perhaps something like this?
 
```suggestion
  format: (val: any) => val instanceof Date ? DateTime.fromJSDate(val).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY) : '',
```

Reply:
Hm, I'm worried this might still be hiding the error, perhaps even more deeply. I think an indication of some kind that the value is bad rather than absent would help.

It'd be nice if the type checks could go work across the component, but that seems like a tall order.

Reply:
So there are 3 states: Date, not Date but truthy (error state), and undefined?

```suggestion
  format: (val: any) => val instanceof Date ? DateTime.fromJSDate(val).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY) : val ? 'Error: non-Date' : '',
```

I tried getting type checks working across the table component (ie applying column defs to row data, so that the `format` function would get a specific type) but retrofitting it was quite hard so I stopped and focused on other stuff. I could make a new table component with a similar API though.

Reply:
A component that made this possible would be neat, but I wouldn't put it as a high priority. What you've got there with the "error" output looks fine to me.